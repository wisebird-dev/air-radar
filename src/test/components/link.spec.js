import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Link from '../../components/link'

test('Link renders correctly', async () => {
  const tree = await renderer.create(<Link />).toJSON()

  expect(tree).toMatchSnapshot()
})