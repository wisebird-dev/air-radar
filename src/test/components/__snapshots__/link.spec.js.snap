// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link renders correctly 1`] = `
.c0 {
  color: #fafff7;
  box-sizing: border-box;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

<a
  className="c0"
/>
`;
